_posts/2019-09-02-eunji.md

```diff
@@ -1,7 +1,7 @@
 layout: contribution
 title: First attendance on October!
 category: meetup
-date:   2019-10-02 00:19:50 +0900
+date:   2019-09-02 00:19:50 +0900
```

Comment:
Original one, 10-02, was right. What you've to change is the file name. Don't worry you can change it.

Reply:
No way!! ;( I think I was confused. Is the file Name means " 2019-10-02-HackerEunji.md"?